src/stepfunctions/steps/choice_rule.py

```diff
@@ -376,6 +592,120 @@ def TimestampGreaterThanEquals(cls, variable, value):
         """
         return Rule(variable, 'TimestampGreaterThanEquals', value)
 
+    @classmethod
+    def TimestampGreaterThanEqualsPath(cls, variable, value):
+        """
+        Creates a rule with the `TimestampGreaterThanEqualsPath` operator.
+
+        Args:
+            variable (str): Path to the variable to compare.
+            value (str): Path to the value to compare `variable` against.
+
+        Returns:
+            Rule: Rule with `TimestampGreaterThanEqualsPath` operator.
+        """
+        return Rule(variable, 'TimestampGreaterThanEqualsPath', value)
+
+    @classmethod
+    def IsNull(cls, variable, value):
+        """
+        Creates a rule with the `IsNull` operator.
+
+        Args:
+            variable (str): Path to the variable to compare.
+            value (bool): Constant value to compare `variable` against.
```

Comment:
For the type comparisons, maybe the documentation for `value` can be more descriptive:

- IsNull: "Whether the value at `variable` is equal to the JSON literal `null` or not.
- IsPresent: "Whether a field at `variable` exists in the input or not.
- IsNumeric: "Whether the value at `variable` is a string or not.
- etc.

Reply:
fixed in subsequent commit